### DISCUSSION TITLE
Long Reads!

### DISCUSSION THREAD

**Discussion by aljazdzy** [franciscozorrilla/thesis_pipeline#152] [Q&A] [answered]
I know this pipeline is best designed for short reads, but would there be ways to optimize it for long-reads beyond changing the assembler/changing assembler settings?

**Comment by franciscozorrilla** [comment, marked as answer]
Great question! I am currently looking into this topic for a project that recently came up, hopefully will have something to share soon. For now, switching out the assembler for something like [flye](https://github.com/fenderglass/Flye) would generate assemblies that could be fed through the metaGEM workflow without any snags. Will make a PR soon 🤞.
